scylla/src/transport/connection.rs

```diff
@@ -91,8 +94,21 @@ impl Connection {
         // TODO: What should be the size of the channel?
         let (sender, receiver) = mpsc::channel(128);
 
-        let (fut, _worker_handle) = Self::router(stream, receiver).remote_handle();
-        tokio::task::spawn(fut);
+        let _worker_handle = match config.use_tls {
+            true => {
+                let mut tls_config = ClientConfig::new();
+                tls_config
+                    .root_store
+                    .add_server_trust_anchors(&webpki_roots::TLS_SERVER_ROOTS);
```

Comment:
We must allow the user to provide their own root certificates to trust via configuration options. E.g. rustls' docs describes how custom root certificates can be loaded from a file: https://docs.rs/rustls/0.1.0/rustls/index.html

We also shouldn't use webpki_roots and rely on this crate at all - the users can provide such certificates themselves if they want to. Instead, the users should be able to provide these anchors explicitly if they want them.